app/cafe.py

```diff
@@ -0,0 +1,20 @@
+from datetime import datetime
+from app import errors
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise errors.NotVaccinatedError("Not vaccine")
+
+        if datetime.date(datetime.now()) > \
```

Comment:
```suggestion
        if date.today() > visitor["vaccine"]["expiration_date"]:
```
Add import 
from datetime import date